File: code_blocks/integrations/third-party-integrations/braze_1.kt
import android.app.Application
import com.braze.Braze
import com.revenuecat.purchases.Purchases

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Purchases.configure(this, "public_sdk_key", appUserID = "my_app_user_id")
        Braze.getInstance(applicationContext).changeUser("my_app_user_id")
        Purchases.sharedInstance.attribution.setAttributes(
            mapOf(
                "\$brazeAliasName" to "name",
                "\$brazeAliasLabel" to "label"
            )
        )
    }
}

File: code_blocks/integrations/third-party-integrations/braze_1.ts
import Purchases from 'react-native-purchases';
import braze from '@braze/react-native-sdk';

Purchases.configure({ apiKey: 'public_sdk_key', appUserID: 'my_app_user_id' });
braze.changeUser('my_app_user_id');

Purchases.setAttributes({
  '$brazeAliasName': 'name',
  '$brazeAliasLabel': 'label',
});
